lexibank_hubercolumbian.py

```diff
@@ -1,83 +1,60 @@
-from itertools import groupby
 from pathlib import Path
 
 import attr
 import lingpy
 from clldutils.misc import slug
-from pylexibank import Concept
+from pylexibank import Concept, Language
 from pylexibank.providers import qlc
 from pylexibank.util import progressbar
 
 
 @attr.s
-class HConcept(Concept):
+class CustomConcept(Concept):
     Spanish = attr.ib(default=None)
     Gloss_in_digital_source = attr.ib(default=None)
 
+@attr.s
+class CustomLanguage(Language):
+    Longitude = attr.ib(default=None)
+    Latitude = attr.ib(default=None)
+    Name_in_Source = attr.ib(default=None)
+
 
 class Dataset(qlc.QLC):
     dir = Path(__file__).parent
     id = "hubercolumbian"
     DSETS = ["huber1992.csv"]
-    concept_class = HConcept
+    concept_class = CustomConcept
```

Comment:
I prefer to have a unified name space here: `Custom*`, this might also help to grep things, etc. It's of course not needed, but makes it clearer, also for newbies.